### ISSUE TITLE
Map Editor Cannot Save Created Maps

### ISSUE THREAD

**Issue by timfel** [Wargus/wargus#105]
After I got it to work thanks to Kyran Jackson, I tinkered with the editor to make some better maps for the game. Turns out the editor cannot save maps that you create with the following error:

"Cannot save *insert map name here* to file *insert path name here*"

I tried putting the map in different locations and even on a different hard drive, and it flat out refuses to save the map.

- Trunk Build (latest 2/7/14)
- Original WarCraft 2 Expansion DOS CD
- stratagus version 2.2.7 (latest)
- wargus version 2.2.7 (latest)
- Windows 7

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2014-02-11T03:14:11Z
 * owner: enkerzan
 * assignee: iddqd-mail
 * the launchpad url was https://bugs.launchpad.net/bugs/1278688

**Comment by timfel** [comment]
(by erstmap)
I can't duplicate this issue.

**Comment by timfel** [comment]
(by erstmap)
"- stratagus version 2.2.7 (latest)"

You should be using the trunk binary I included in the zip. How do you like the direction the trunk version is taking?

**Comment by timfel** [comment]
(by enkerzan)
Everything I am running is from the .ZIP I got from you and on a fresh install of the latest Wargus/Stratagus version that can be downloaded. Not sure why it isn't saving the maps, there is another method I can try to use to see if it's a Windows 7 issue...

Overall this trunk build works much better than the previous versions, although the AI is still a little questionable, but more playable against than before. Will have to test further when I can get the map editor working right...

**Comment by timfel** [comment]
(by enkerzan)
OK, I found the problem.

For some reason this trunk build conflicts Window 7's default security settings, meaning it will not create new files within the operating system hard drive, nor will it allow you or the game to edit any files.

Kind of strange considering it didn't do that before...

A couple of small things I noticed with the Map Editor:

1. Loading a map from the main menu screen seems to be locked into the skirmish folder and won't allow you to browse other files.

2. The music doesn't cycle between all of the game's tracks, it only picks Alliance only or Horde only tracks. It always starts with Horde music first, til you decide to go back into the editor again, then it switches to Alliance. Gets kind of annoying hearing the Horde music loop over and over, it's like an endless Zales Jewelry commercial that you cannot mute without shutting your sound off. lol
